routes/users.js

```diff
@@ -629,28 +629,10 @@ router.get('/dashboard2/:username/mytapplications', function(req, res){
 });
 
 router.get('/calendar', loggedIn , function(req, res){
-	res.render('calendar');
+	res.render('calendar',{username : req.params.username, h1 : req.user.holidays, h2 : req.user.halfdays , h3 : req.user.nonfunc_holidays, flag : req.user.flag });
```

Comment:
why are we sending username? Also, req.params is when we take parameters from url string, username isn't there in calendar url.